.github/workflows/turborepo-release.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         uses: ./.github/actions/setup-turborepo-environment
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
-          target: ${{ matrix.os.name }}
+          target: ubuntu-latest
```

Comment:
Maybe change this to `linux` instead of `ubuntu-latest`? `ubuntu-latest` gives the impression that this target should be a GitHub Action Runner when that's incorrect.

Out of scope for this PR, but we might want to change this argument to just be `windows` boolean flag. Currently it only gets passed to `setup-rust` and it only has a behavior change if it is set to [`windows`](https://github.com/vercel/turbo/blob/main/.github/actions/setup-rust/action.yml#L51) (which isn't a [GH Runner string](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources))